apis/core/lib/domain/csv-mapping/CsvMapping.ts

```diff
@@ -1,11 +1,14 @@
 import { Constructor } from '@tpluscode/rdfine'
 import { rdf } from '@tpluscode/rdf-ns-builders'
 import { cc } from '@cube-creator/core/namespace'
-import { CollectionMixin } from '@rdfine/hydra'
+import * as Hydra from '@rdfine/hydra'
+import RdfResource from '@tpluscode/rdfine/RdfResource'
 import { CsvMapping } from '@cube-creator/model'
 import { ResourceStore } from '../../ResourceStore'
 import * as id from '../identifiers'
 
+RdfResource.factory.addMixin(...Object.values(Hydra))
```

Comment:
This was a missing piece which is a major pitfall and PITA with rdfine when you forget to set up the resource factory